spec/views/home/index.html.slim_spec.rb

```diff
@@ -3,11 +3,13 @@
 RSpec.describe 'home/index.html.slim', type: :view do
   describe 'time_entries' do
     it 'it shows the data of a user' do
-      assign(:final_entries, [{ 'name' => 'Max Muster', 'open_investment_time' => 1 }])
+      assign(:employee, [{ 'name' => 'Max Muster', 'open_investment_time' => 1,
+                           'worked_total_hours' => 10, 'worked_investment_hours' => 0 }])
 
       render
 
       expect(rendered).to include 'Max Muster'
+      expect(rendered).to include '1'
```

Comment:
I guess, you can come up with a criteria which is a bit more precise than just searching for a `1` ;-)